common/api/imodeljs-backend.api.md

```diff
@@ -4001,7 +4003,7 @@ export class SqliteStatement implements IterableIterator<any>, IDisposable {
     get isPrepared(): boolean;
     get isReadonly(): boolean;
     next(): IteratorResult<any>;
-    prepare(db: IModelJsNative.DgnDb | IModelJsNative.ECDb | IModelJsNative.SQLiteDb): void;
+    prepare(db: IModelJsNative.DgnDb | IModelJsNative.ECDb | IModelJsNative.SQLiteDb, logErrors?: boolean): void;

```

Comment:
Here and elsewhere should use `logErrors = true`. People generally expect `undefined` for a boolean arg means `false`.

Reply:
In code its set to default true. The extract api remove the default arguments.